timestamp-task/README.md

```diff
@@ -0,0 +1,26 @@
+Timestamp Job
+============================
```

Comment:
Can we use asciidoctor for this please?
Also, please do not use `=======` underlying for titles, but rather prefixes. They're easier to change via `sed` or similar if needed